lib/libzpool/uzfs_zap.c

```diff
@@ -50,6 +51,17 @@ uzfs_update_zap_entries(void *zvol, const uzfs_zap_kv_t **array,
 	int err;
 	int i = 0;
 
+	/*
+	 * check if key length is greater than MZAP_NAME_LEN.
+	 * key with MZAP_NAME_LEN+ length will convert microzap
```

Comment:
Thanks @mynktl for these changes, so that, it won't become fat zap. Comment need to be fixed at this line.

Reply:
ok